wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
 |**Genre Classical/Folk** |`classical.jamulus.io:22524`|
 |**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
 
-You can also specify a [Directory](#3-directory) in the same way from the command line, providing the Server Address in the same format.
 
-#### Running as a Directory
+##### `-o or --serverinfo` 
 
-If you wish to run a [Directory](Running-a-Server#3-directory) please see [this guide](Directories).
+When registering a server on a Directory, this lets you display a server name and location details so that users can then search or sort by these values when connecting to Directories with their client.
```

Comment:
```suggestion
When registering a server with a Directory, this lets you display a server name and location details so that users can then search or sort by these values when connecting to Directories with their client.
```